src/webui/webui.h

```diff
@@ -32,6 +32,7 @@
 #include <QObject>
 #include <QPointer>
 
+#include "base/3rdparty/expected.hpp"
```

Comment:
seems you don't need this.

Reply:
> seems you don't need this.

You're right. This is a forgotten remnant from one of the previous implementations. Deleted it.